server/controllers/removeCellType/check.test.ts

```diff
@@ -23,7 +23,7 @@ describe('CheckRemoveCellType', () => {
           areYouSure: 'yes',
         },
       },
-    }
+    } as unknown as FormWizard.Request
```

Comment:
We could just do `req: FormWizard.Request = ...` and add a `@ts-ignore` above it, or if you prefer to do it this way then we can probably remove the other `@ts-ignore`s above.

Reply:
I did this because IDEA was still shouting at me with the ts-ignores in, but also it means we just need this 1 line instead of multiple @ts-ignore throughout the object